File: library/src/main/java/com/custom/photoView/photoTap/GestureDetector.java
package com.custom.photoView.photoTap;

import android.view.MotionEvent;

import com.custom.photoView.photoTap.OnGestureListener;

public interface GestureDetector {

    boolean onTouchEvent(MotionEvent ev);

    boolean isScaling();

    boolean isDragging();

    void setOnGestureListener(OnGestureListener listener);

}

File: library/src/main/java/com/custom/photoView/photoTapListener/GestureDetector.kt
package com.custom.photoView.photoTapListener

import android.view.MotionEvent

import com.custom.photoView.photoTapListener.OnGestureListener

interface GestureDetector {

    abstract fun onTouchEvent(ev: MotionEvent): Boolean

    abstract fun isScaling(): Boolean

    abstract fun isDragging(): Boolean

    abstract fun setOnGestureListener(listener: OnGestureListener)

}
